src/Nancy/Responses/DefaultJsonSerializer.cs

```diff
@@ -100,8 +100,7 @@ private static bool IsJsonType(string contentType)
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||

```

Comment:
According to [this](http://www.iana.org/assignments/media-types/media-types.xhtml#application) `application/json-patch+json` and `application/json-seq` are valid media types. There are some similar XML ones listed too. Could simplify the checks to just `contentMimeType.StartsWith("application/json", StringComparison.OrdinalIgnoreCase)` ?

I'll fix the formatting...